### PR TITLE
Use GitHub App to push version bumps

### PR DESCRIPTION
**Pull request by Felixmil**
TO DO:

- [x] Create an organization GitHub App (with write access to repo content) + private key
- [x] Add App ID and private key as organization variables (`ACTIONS_MANAGER_APPID` and `ACTIONS_MANAGER_TOKEN`)
- [x] Install App for OSP repositories
- [x] Restore review requirement before merging

### CODE REVIEW

File: .github/workflows/main-workflow.yaml
Comment (Yuri05) [inline .github/workflows/main-workflow.yaml:1]: now that we don't have the **develop** branch anymore - it can be deleted from the trigger conditions

Comment (Yuri05) [inline .github/workflows/main-workflow.yaml:1]: Question: how do we prevent that the commit executed by **bump-dev-version** prevents another **on push** workflow triggering?

File: .github/workflows/main-workflow.yaml [hunk 1]
```diff
@@ -11,6 +11,8 @@ jobs:
   bump-dev-version: # only do that when actually merging in main/develop branch
     if: github.event_name != 'pull_request'
     uses: ./.github/workflows/bump-dev-version.yaml
+    secrets: inherit
```
Comment (Yuri05) [inline .github/workflows/main-workflow.yaml:14]: This inherits the secrets from the organization?

Comment (Felixmil) [reply .github/workflows/main-workflow.yaml:14]: Yes

File: .github/workflows/main-workflow.yaml
Comment (Felixmil) [reply .github/workflows/main-workflow.yaml:1]: By adding `[skip actions]` in the commit message added by the action.

Comment (Felixmil) [reply .github/workflows/main-workflow.yaml:1]: done in 3ab28a8

Comment (Yuri05) [reply .github/workflows/main-workflow.yaml:1]: Nice. Didn't know it works like this.

File: .github/workflows/main-workflow.yaml (hunk 1 above)
Comment (Yuri05) [reply .github/workflows/main-workflow.yaml:13]: _Environment variables_ from the organization are inherited automatically?

**Review by Yuri05 — APPROVED** [review]

File: .github/workflows/main-workflow.yaml (hunk 1 above)
Comment (Felixmil) [reply .github/workflows/main-workflow.yaml:13]: Yes:
> To reuse variables in multiple workflows, set them at the organization, repository, or environment levels and reference them using the vars context. For more information see "[Variables](https://docs.github.com/en/actions/learn-github-actions/variables)" and "[Contexts](https://docs.github.com/en/actions/learn-github-actions/contexts#vars-context)."

from [here](https://docs.github.com/en/actions/using-workflows/reusing-workflows#limitations)

**Pull request merged**